### DISCUSSION TITLE
What is the difference between images?

### DISCUSSION THREAD

**Discussion by life8853** [Joshua-Riek/ubuntu-rockchip#152] [Q&A] [answered]
So i am looking forward to buying a orange pi 5 8gb (my first SBC) and I'm very keen on the hardware (since you can't really buy a RPi). While doing research on the software compatibility side I stumbled on a question that i can't really solve.

What is the difference between this ubuntu image, the one on http://www.orangepi.org/ (that you have to sketchly download from Google Drive) and the ARM image directly from ubuntu.com.

Thanks from advance for any help! I also think that putting the answer on the github wiki or readme would be very useful to people.

**Comment by Joshua-Riek** [comment, marked as answer]
The ARM image directly from ubuntu.com will not work with this device. The goal of my image is to be as close as possible to Ubuntu which can be found on the Rasberry Pi, including a setup wizard for some user setup and configuration. There will be some differences

The Ubuntu image available from Orange Pi is an old fork of Armbian with a lot of broken software, drivers, pre-defined user account, and no way to distribute kernel / firmware updates to users.
